packages/public/server/src/module/Ui/templates/entity/unrevised.twig

```diff
@@ -0,0 +1,57 @@
+{#
+ # This file is part of Serlo.org.
+ #
+ # Copyright (c) 2013-2019 Serlo Education e.V.
+ #
+ # Licensed under the Apache License, Version 2.0 (the "License")
+ # you may not use this file except in compliance with the License
+ # You may obtain a copy of the License at
+ #
+ #    http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+ #
+ # @copyright Copyright (c) 2013-2019 Serlo Education e.V.
+ # @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ # @link      https://github.com/serlo-org/serlo.org for the canonical source repository
+ #}
+{{ pageHeader('Unrevised learning resources').render() }}
+<div class="unrevised">
+{% for subject, revisionsByEntity in revisionsBySubject %}
+  <h2>{{ subject }}</h2>
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <th>{% trans %} Title {% endtrans %}</th>
+        <th>{% trans %} Revision {% endtrans %}</th>
+        <th>{% trans %} Type {% endtrans %}</th>
+        <th>{% trans %} Timestamp {% endtrans %}</th>
+      </tr>
+    </thead>
+    <tbody>
+    {% for revisions in revisionsByEntity %}
+        {% for revision in revisions %}
+          <tr>
+            <td style="width:30%">
+              {% if loop.first %}
+                {{ revision.getTitle() }}
+              {% endif %}
+            </td>
+            <td>
+              <a href="{{ url(revision.getRouteName(), revision.getRouteParams()) }}">
```

Comment:
#137 asked if we could open this in a new tab
```suggestion
              <a href="{{ url(revision.getRouteName(), revision.getRouteParams()) }}" target="_blank">
```

Reply:
Not the biggest fan of forcing that IMHO. Can't users that want to open it in a new tab do that themselves? E.g. by Ctrl-clicking or by clicking with the middle mouse button.

Reply:
Well, that might not be known to the users, so I think we have to discuss that with @yolo-bird.
The issue says: 
>Zur Überprüfung der Bearbeitung öffnet sich ein neues Fenster. Dadurch muss man nicht wieder zur Übersichtsseite der ungeprüften Bearbeitungen gehen."

Reply:
It's ok for me to let the users decide this.